cmd/initContainer/main.go

```diff
@@ -119,8 +121,63 @@ func main() {
 	failure := false
 
 	run := func() {
-		_, err := kubeClientLeaderElection.CoordinationV1().Leases(getKyvernoNameSpace()).Get(ctx, "kyvernopre-lock", v1.GetOptions{})
+		certProps, err := tls.GetTLSCertProps(clientConfig)
```

Comment:
Usually we check the error first, and proceed with the rest logic if `err==nil`. This way it's easier for others to read the code.

For example:

```code
err := get()
if err != nil{
  // handle the error
  // return/exit
}

// main logic
```